tests/redirects/map_globalconf.py

```diff
@@ -938,5 +942,28 @@
 
     # bug 1238248
     url_test('/firefox/push/',
-             'https://support.mozilla.org/kb/push-notifications-firefox')
+             'https://support.mozilla.org/kb/push-notifications-firefox'),
+
+    # from mcom-tests
+    url_test('/firefox/', '/en-US/firefox/'),
+    url_test('/en-US/firefox/', '/firefox/new/'),
+    url_test('/firefox/new/', '/en-US/firefox/new/'),
+    url_test('/fennec/', '/firefox/partners/'),
+    url_test('/firefox/mobile/', '/firefox/android/'),
+    url_test('/mobile/37.0{,beta,a2}/releasenotes', '/firefox/android/37.0{,beta,a2}/releasenotes/'),
+    url_test('/projects/firefox/3.6.13/whatsnew/', '/firefox/3.6.13/whatsnew/'),
+    url_test('/en-US/firefox/3.6.13/{firstrun,whatsnew}/', '/en-US/firefox/new/'),
+    url_test('/b2g/', '/firefox/partners/'),
```

Comment:
Note: this test will fail once #3781 merges (and also adds an equivalent redirect test). Will just need to rebase depending on whichever PR merges first.